src/tiled/mapdocument.cpp

```diff
@@ -335,6 +335,19 @@ void MapDocument::resizeMap(const QSize &size, const QPoint &offset, bool remove
     // TODO: Handle layers that don't match the map size correctly
 }
 
+
+void MapDocument::autocropMap()
+{
+    // Since autocrop option is allowed only for Tile Layer
+    TileLayer* tileLayer = static_cast<TileLayer*>(mCurrentLayer);
+
+    const QRect bounds = tileLayer->computeAutocropRegion().boundingRect();
+    if (bounds.isNull())
+        return;
+
+    resizeMap(bounds.size(), -bounds.topLeft(), true);
```

Comment:
Yes, I think it's reasonable for this to be consistent with "Crop to Selection". There isn't really an opportunity to ask the user here, unless we'd introduce a popup, which I don't really want to.